src/style.css

```diff
@@ -255,6 +255,41 @@ textarea {
   align-items: center;
 }
 
+@media only screen and (max-width: 600px) {
+  #gradient-text-color1 {
+    width: 300px;
+    height: 25px;
+  }
+  #gradient-text-color2 {
+    width: 300px;
+    height: 25px;
+  }
+  #gradient-border-color1 {
+    width: 300px;
+    height: 25px;
+  }
+  #gradient-border-color2 {
+    width: 300px;
+    height: 25px;
+  }
+  #gradient-background-color1 {
+    width: 300px;
+    height: 25px;
+  }
+  #gradient-background-color2 {
+    width: 300px;
+    height: 25px;
+  }
+  #box-shadow-color {
+    width: 300px;
+    height: 25px;
+  }
+  #text-shadow-color {
+    width: 300px;
+    height: 25px;
+  }
+}
```

Comment:
You can do all this in one line, using the data-coloris attribute available to style in CSS

this is what it looks like in HTML
https://github.com/Dun-sin/Code-Magic/blob/f6d319500ce3063a9bebfaae25133ccb53fd827b/index.html#L245

Reply:
Also use dynamic with and not this, we already called the width for generators

use this variable for width
https://github.com/Dun-sin/Code-Magic/blob/f6d319500ce3063a9bebfaae25133ccb53fd827b/src/style.css#L14

Reply:
> Also use dynamic with and not this, we already created the width for generator's input, and it doesn't need to be in media query
> 
> use this variable for width
> 
> https://github.com/Dun-sin/Code-Magic/blob/f6d319500ce3063a9bebfaae25133ccb53fd827b/src/style.css#L14

this variable is not working properly. because the inputs are in the "color" div , so the 80% of width is getting outside of the screen

Reply:
> > Also use dynamic with and not this, we already created the width for generator's input, and it doesn't need to be in media query
> > use this variable for width
> > https://github.com/Dun-sin/Code-Magic/blob/f6d319500ce3063a9bebfaae25133ccb53fd827b/src/style.css#L14
> 
> this variable is not working properly. because the inputs are in the "color" div , so the 80% of width is getting outside of the screen

You can create a width using the clamp, having a fixed width is worse than having a little bug

Reply:
You just need to align it properly using the media query
![image](https://user-images.githubusercontent.com/78784850/197980170-34065bf8-5ae1-4dce-a0d3-4217a19696e4.png)

Reply:
@Dun-sin can i make another variable ?

Reply:
> @Dun-sin can i make another variable ?

no need really, you only write this once anyway